### PR TITLE
Json-result MVC example

### PR DESCRIPTION
**Pull request by genesisrrios**
Hi, I wrote an MVC JSON result example.
If I'm missing anything or something is unclear let me know and I will gladly fix it.

### CODE REVIEW

File: projects/mvc/result-json/result-json.csproj [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <AssemblyName>result</AssemblyName>
```
Comment (dodyg) [inline projects/mvc/result-json/result-json.csproj]: Remove these properties. They aren't needed,

File: projects/mvc/result-json/src/Program.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Hosting;
+using System;
+
+namespace PracticalAspNetCore
+{
+	public class Startup
+	{
+		public void ConfigureServices(IServiceCollection services)
```
Comment (dodyg) [inline projects/mvc/result-json/src/Program.cs:12]: Use 4 spaces tabs please

File: projects/mvc/result-json/src/Program.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+	}
+
+	public class HomeController : Controller 
+	{
+		IWebHostEnvironment _env;
+
+		public HomeController(IWebHostEnvironment env)
+		{
+			_env = env;
+		}
```
Comment (dodyg) [inline projects/mvc/result-json/src/Program.cs:35]: Please create an index page so people can just run it and see a page. In this index page, put a one paragraph description and link it to the endpoint that return json.

File: projects/mvc/result-json/result-json.csproj (hunk 1 above)
Comment (dodyg) [reply projects/mvc/result-json/result-json.csproj]: Remove everything except TargetFramework

Comment (genesisrrios) [reply projects/mvc/result-json/result-json.csproj]: Removed the unnecessary properties.

File: projects/mvc/result-json/src/Program.cs (hunk 2 above)
Comment (genesisrrios) [reply projects/mvc/result-json/src/Program.cs:12]: The spaces should be fixed now

File: projects/mvc/result-json/src/Program.cs (hunk 3 above)
Comment (genesisrrios) [reply projects/mvc/result-json/src/Program.cs:35]: Added the index page in commit 46552b7

**Pull request merged**